### ISSUE TITLE
[HOLD][$500] IOS - Money request- Keyboard displayed for add receipt

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#28163]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing PR** https://github.com/Expensify/App/pull/27224
## Action Performed:
1. Navigate to + > Request money
2. Select Manual and go through the flow without adding a receipt
3. Once the request is added to the report, tap the preview twice so you can see the transaction details
4. Tap the top three dot menu and select Add receipt

## Expected Result:
Option to take picture or upload pictures

## Actual Result:
Keyboard also opens

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [x] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.74.1

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/047c838d-313f-4fa6-bd8d-49d02acb90f9

![Bug6213778_IMG-7468](https://github.com/Expensify/App/assets/78819774/2ce195a1-17d1-432d-849d-599ff31d03e4)

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f0b3e2e150e7efad</li>
        <li>Upwork Job ID: 1706752558346231808</li>
        <li>Last Price Increase: 2023-10-03</li>
<li>Automatic offers: </li>
<ul>
<li>0xmiroslav | Reviewer | 27105736</li>
<li>tienifr | Contributor | 27105740</li>
</ul></li>
    </ul>
</details>

**Comment by NicMendonca** [comment]
> Keyboard also opens

I don't think these needs to be an hourly 😅 

@chiragsalian should I downgrade this to daily?

**Comment by chiragsalian** [comment]
So because it is reproducible on staging and not on production its an hourly. Now we need to decide wether we need to CP code to fix this or mark as non deploy blocker first before we can demote it. I haven't investigated the issue enough yet, not sure if i will get time for it today.

**Comment by chiragsalian** [comment]
Yup, i was just explaining to nicole why QA would label this as hourly deployblocker.

Investigated today. Its definitely not a valid deploy blocker since its a minor bug IMO and can be fixed easily. Demoting to daily and applying external label.

**Comment by MichaelAwinda** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The keyboard is also displayed for add receipt, instead of giving an option for taking a picture or uploading a picture.
### What is the root cause of that problem?
The receipt input field is using a numeric keyboard type, which does not allow the user to take a photo or choose an image from the device.
### What changes do you think we should make in order to solve the problem?

The add receipt input field should use a default keyboard type, which will show a normal keyboard with an option to access the camera or gallery. The user will be able to select an image or file for the receipt.
### What alternative solutions did you explore? (Optional)

**Comment by MichaelAwinda** [comment]
Contributor details
Your Expensify account email: michaelawinda1@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01d2d01066fd216588

**Comment by NicMendonca** [comment]
@0xmiroslav proposals for you! ^^

**Comment by NicMendonca** [comment]
@0xmiroslav bump bump ^

**Comment by NicMendonca** [comment]
@0xmiroslav feedback? ^^

**Comment by chiragsalian** [comment]
Proposal LGTM, feel free to create the PR @tienifr.

**Comment by NicMendonca** [comment]
@tienifr @0xmiroslav any update on this one?

**Comment by chiragsalian** [comment]
Yeah i think your comment [here](https://github.com/Expensify/App/pull/29255#issuecomment-1818405509) makes sense to me and we should put this on hold.

**Comment by twisterdotcom** [comment]
What should I be doing here now @chiragsalian?

**Comment by chiragsalian** [comment]
For you ted, nothing atm.
For @0xmiroslav / @tienifr, i believe this issue can be off HOLD now that the linked issues are completed. Can you confirm? 
Then @tienifr, are you still interested in working on this, if so let us know your proposal.

**Comment by chiragsalian** [comment]
Yup looks like this is non-issue now, closing.

**Comment by twisterdotcom** [comment]
Okay yeah, looks like @0xmiroslav was paid already in https://www.upwork.com/nx/wm/workroom/35369345/overview, so will pay out @tienifr here too.

Payment Summary:

- @0xmiroslav paid $500 via Upwork [here](https://www.upwork.com/nx/wm/workroom/35369345/overview) (C+)
- @tienifr paid $500 via Upwork [here](https://www.upwork.com/nx/wm/workroom/35094596/overview) (Contributor)

I know you're on manual requests now but as this is an old issue, we have to still pay them via Upwork @tienifr.
